src/Sylius/Bundle/CoreBundle/Provider/Processable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Provider;
+
+interface Processable
```

Comment:
```suggestion
interface VariantsProviderInterface
```
and there is missing `provideEligibleVariants` method